src/components/BorrowerProfile/LendCta.vue

```diff
@@ -749,8 +749,8 @@ export default {
 		isLessThan25() {
 			return this.unreservedAmount < 25 && this.unreservedAmount > 0;
 		},
-		isBetween25And100() {
-			return this.unreservedAmount < 100 && this.unreservedAmount > 25;
+		isBetween25And75() {
+			return this.unreservedAmount < 75 && this.unreservedAmount > 25;
```

Comment:
Do we want this to include `75` and `25`?

Reply:
nope exclusive!